orderer/common/multichannel/registrar.go

```diff
@@ -516,6 +516,45 @@ func (r *Registrar) SwitchFollowerToChain(chainName string) {
 	logger.Infof("Created and started channel %s", cs.ChannelID())
 }
 
+// SwitchChainToFollower creates a follower.Chain from the tip of the ledger and removes the consensus.Chain.
+// It is called when an etcdraft.Chain detects it was evicted from the cluster (i.e. removed from the consenters set)
+// and halts, transferring execution to the follower.Chain.
+func (r *Registrar) SwitchChainToFollower(channelName string) {
+	r.lock.Lock()
+	defer r.lock.Unlock()
+
+	if _, chainExists := r.chains[channelName]; !chainExists {
+		logger.Infof("Channel %s already removed", channelName)
```

Comment:
Perhaps it would be clearer to say "Channel %d consenter is not active (is a follower, or was removed)" or similar?

Reply:
Done